src/main/java/com/icthh/xm/uaa/service/account/password/reset/type/custom/CustomPasswordResetHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.icthh.xm.uaa.service.account.password.reset.type.custom;
+
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import com.icthh.xm.commons.lep.spring.LepService;
+import com.icthh.xm.uaa.service.account.password.reset.PasswordResetHandler;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@LepService(group = "service.account.password.reset")
```

Comment:
All other password reset functionality now defined in the group `group = "service.user"`
I think it worth to use `service.user` group here as well. 
@ishkurko what do you think?

Reply:
I checked previous reset password leps and indeed, all have group "service.user" so I changed the group either